src/main/java/atmosfears/AtmosFearsBE/model/Predictor.java

```diff
@@ -0,0 +1,66 @@
+package atmosfears.AtmosFearsBE.model;
+
+import com.github.rcaller.rstuff.RCaller;
+import com.github.rcaller.rstuff.RCode;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class Predictor {
+
+    public static double[] predict(String csv_file_path){
```

Comment:
Maybe it will be better to return List object not a primitve array? I don't really know how it will be used, but using List are much easier and clearer that arrays